src/components/ImageComponent/image-component.scss

```diff
@@ -46,9 +46,17 @@
 
   img.gatsby-resp-image-image {
     max-width: 736px;
-    width: 736px !important;
+    width: 100%;
     height: auto !important;
     position: relative !important;
+
+    @include breakpoint('sm') {
+      min-width: 368px !important;
```

Comment:
We want to keep the images 736px as long as possible, and with this added code the image shrinks very early on. I wonder if we can instead to a min-width only on mobile screens?